conf/messages.en

```diff
@@ -608,6 +608,44 @@ view.amend-case.summary.documents.heading=Documents you are submitting
+view.beta.paragraph2=You only need to send documents to HMRC if your declaration status shows a requirement for documents to be sent.
+view.beta.paragraph3=Customs clearance routes include:
+view.beta.routes.route1=Route 1: a full documentary check
+view.beta.routes.route1cap=Route 1 CAP (Common Agricultural Policy)
+view.beta.routes.route2=Route 2: a physical examination of both your goods and documents
+view.beta.routes.route3=Route 3: your goods can get cleared, but you must present your documents within 24 hours
+view.beta.routes.route6=Route 6: instant clearance where you do not need to present any documents
+view.beta.paragraph4=You can also use this service to send C1601, C1602 and C1603 forms to report exports that arrived or left a UK port that were not declared in CHIEF.
+view.beta.links.c1601=C1601 - Presentation of goods for export (arrival)
+view.beta.links.c1602=C1602 - Notification of exit of goods (departure)
+view.beta.links.c1603=C1603 - Notification of retrospective arrival
+view.beta.whatYouNeed=What you’ll need
+view.beta.whatYouNeed.paragraph1=You’ll need to upload all the paperwork that supports your declaration, for example:
+view.beta.whatYouNeed.item1=any licences or certificates
+view.beta.whatYouNeed.item2=evidence of the goods’ value
+view.beta.whatYouNeed.item3=a packing list of the items included
+view.beta.whatYouNeed.item4=proof of origin certificates
+view.beta.howToUpload=How to upload documents
+view.beta.howToUpload.paragraph1=You can upload up to 10 documents, each with a maximum file size of 6MB.
+view.beta.howToUpload.paragraph2=These file types are allowed:
+view.beta.howToUpload.fileTypes1=image (.jpg, .jpeg, .png or .tiff)
+view.beta.howToUpload.fileTypes2=PDF (.pdf)
+view.beta.howToUpload.fileTypes3=email (.txt, .oft or .msg)
+view.beta.howToUpload.paragraph3=You cannot upload Microsoft files, zip files or documents that are password-protected.
+view.beta.howToUpload.paragraph4=If you have a Word or Excel document to upload, you need to convert it to PDF first.
+view.beta.howToUpload.paragraph5=If you’ve already sent your documents for clearance and need to add additional information, you must have the case reference number. You will have received this number when the case was first created.
+view.beta.beforeYouStart=Before you start
+view.beta.beforeYouStart.paragraph1=You will need a user ID and password. If you do not have a user ID, you can create one the first time you send us your documents for customs clearance.
+view.beta.beforeYouStart.startNow=Start now
+view.beta.afterSendingDocuments=After you’ve sent HMRC supporting documents
+view.beta.afterSendingDocuments.paragraph1=After you have completed the service you'll be told when your goods should be cleared. You’ll be notified through your software when they have cleared.
```

Comment:
First apostrophe needs to be a smart one

Reply:
:+1: